meta-rinato/recipes-kernel/samsung-smartwatch-firmware/samsung-smartwatch-firmware_master.bb

```diff
@@ -0,0 +1,35 @@
+SECTION = "kernel"
```

Comment:
The proper filename would be `samsung-smartwatch-firmware_git.bb`

Reply:
Fixed.